scripts/install_git_crypt.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+set -eu
+
+_main() {
+  apk add --update --no-cache curl make g++ libressl libressl-dev
```

Comment:
As opposed to the official git resource, I've chosen to add git-crypt specific dependencies here, because it felt more self-contained.